lib/item.py

```diff
@@ -33,17 +33,17 @@
 from lib.constants import (ITEM_DEFAULTS, FOO, KEY_ENFORCE_UPDATES, KEY_CACHE, KEY_CYCLE, KEY_CRONTAB, KEY_EVAL,
                            KEY_EVAL_TRIGGER, KEY_NAME,KEY_TYPE, KEY_VALUE, PLUGIN_PARSE_ITEM,
                            KEY_AUTOTIMER,KEY_THRESHOLD,
-                           ATTRIB_COMPAT_V12, ATTRIB_COMPAT_LATEST)
+                           KEY_ATTRIB_COMPAT, ATTRIB_COMPAT_V12, ATTRIB_COMPAT_LATEST)
 
 
-# TODO: Decide, if ATTRIB_COMPAT_DEFAULT should be moved to constants.py
-# TODO: Implement reading global setting for ATTRIB_COMPAT_DEFAULT from smarthome.yaml
-ATTRIB_COMPAT_DEFAULT = ATTRIB_COMPAT_V12
+ATTRIB_COMPAT_DEFAULT_FALLBACK = ATTRIB_COMPAT_V12
+ATTRIB_COMPAT_DEFAULT = ''
```

Comment:
The constants are already defined in canstants.py ATTRIB_COMPAT_DEFAULT is no constant. It ist changed through initialization (look at lines 261 - 271).